addons/dataverse/views.py

```diff
@@ -150,16 +150,8 @@ def dataverse_set_config(node_addon, auth, **kwargs):
 @must_have_addon(SHORT_NAME, 'node')
 def dataverse_get_datasets(node_addon, **kwargs):
     """Get list of datasets from provided Dataverse alias"""
-    alias = request.json.get('alias')
-
-    connection = client.connect_from_settings(node_addon)
-    dataverse = client.get_dataverse(connection, alias)
-    datasets = client.get_datasets(dataverse)
-    ret = {
-        'alias': alias,  # include alias to verify dataset container
-        'datasets': [{'title': dataset.title, 'doi': dataset.doi} for dataset in datasets],
-    }
-    return ret, http.OK
+    node_addon.dataverse_alias = request.json.get('alias')
+    return {'alias': node_addon.dataverse_alias, 'datasets': node_addon.get_folders()}, http.OK
```

Comment:
Minor: probably don't need to define node_addon.dataverse_alias here, can just call request.json.get('alias') where you are returning this dictionary? Or is there a reason why you are saving a dataverse alias under node_addon?